public/components/trace_analytics/components/traces/trace_view_helpers.tsx

```diff
@@ -58,7 +58,16 @@ export function getOverviewFields(parsed: any, mode: string) {
 
     const computedLastUpdated = traceGroupFields?.endTime || firstSpan.endTime || null;
     const tgStatus = traceGroupFields?.statusCode;
-    const errorCount = tgStatus != null ? (Number(tgStatus) === 2 ? 1 : 0) : 0;
+    let errorCount = tgStatus != null ? (Number(tgStatus) === 2 ? 1 : 0) : 0;
+
+    // If no trace group error found, check individual spans for errors
```

Comment:
May be we can take care of two formats right after the requests of `get trace` is received in the `traces_request_handler` ?

This way we don't have to check for edge cases all over the codebase

Reply:
Will take this as follow-up only have the two occurrences currently but need to take time to verify central change doesn't break any work-flows.